cmd/test/cmd.go

```diff
@@ -70,10 +70,8 @@ type builder interface {
 func Test(ctx context.Context, ioCtrl *io.Controller, k8sLogger *io.K8sLogger, at *analytics.Tracker) *cobra.Command {
 	options := &Options{}
 	cmd := &cobra.Command{
-		Use:          "test",
-		Short:        "Run tests",
-		Hidden:       true,
-		SilenceUsage: true,
+		Use:   "test",
+		Short: "Run tests defined in your okteto manifest",
```

Comment:
@codyjlandstrom Anything special we want to show in the contextual help of the command? 

This is how it looks like when you run `okteto test --help`

```
Run tests defined in your okteto manifest

Usage:
  okteto test [flags]

Flags:
  -c, --context string     context where the development environment is deployed
      --deploy             Always deploy the dev environment. If it's already deployed it will be redeployed
  -f, --file string        path to the okteto manifest file
  -h, --help               help for test
      --name string        name of the development environment name to be deployed
  -n, --namespace string   overwrites the namespace where the development environment is deployed
      --no-cache           Do not use cache for running tests
  -t, --timeout duration   the length of time to wait for completion, zero means never. Any other values should contain a corresponding time unit e.g. 1s, 2m, 3h  (default 5m0s)
  -v, --var stringArray    set a variable (can be set more than once)

Global Flags:
  -l, --log-level string    amount of information outputted (debug, info, warn, error) (default "warn")
      --log-output string   output format for logs (tty, plain, json) (default "tty")
```

This is how it looks like when you execute `okteto --help`:

```
Okteto - Remote Development Environments powered by Kubernetes

Usage:
  okteto [command]

Available Commands:
  analytics       Enable / Disable analytics
  build           Build and push the images defined in the 'build' section of your okteto manifest
  completion      Generate the autocompletion script for the specified shell
  context         Set the default context
  deploy          Execute the list of commands specified in the 'deploy' section of your okteto manifest
  destroy         Destroy everything created by the 'okteto deploy' command
  doctor          Generate a zip file with the okteto logs
  down            Deactivate your development container
  endpoints       Show endpoints for an environment
  exec            Execute a command in your development container
  help            Help about any command
  init            Automatically generate your okteto manifest
  kubeconfig      Download credentials for the Kubernetes cluster selected via 'okteto context'
  logs            Fetch the logs of your development environment
  namespace       Configure the current namespace of the okteto context
  pipeline        Pipeline management commands
  preview         Preview environment management commands
  status          Status of the synchronization process
  test            Run tests defined in your okteto manifest
  up              Deploy your development environment
  update          Update Okteto CLI version
  version         View the version of the okteto binary

Flags:
  -h, --help                help for okteto
  -l, --log-level string    amount of information outputted (debug, info, warn, error) (default "warn")
      --log-output string   output format for logs (tty, plain, json) (default "tty")

Use "okteto [command] --help" for more information about a command.
```

Reply:
What do you think about making the description: `Run unit and e2e tests defined in your okteto manifest`?

We need to fix the capitalization so that each of the lines starts capitalized

outputted -> output

Reply:
Changed description and modified capitalization of command options to unify it. Change also `outputted`. I also added a link to the docs for the test command